apps/server/src/user/user.controller.ts

```diff
@@ -24,12 +23,12 @@ export class UserController {
   }
 
   @Patch(':id')
-  update(@Param('id') id: ObjectId, @Body() updateUserDto: UpdateUserDto) {
+  update(@Param('id') id: string, @Body() updateUserDto: UpdateUserDto) {
     return this.userService.update(id, updateUserDto);
   }
 
   @Delete(':id')
-  remove(@Param('id') id: ObjectId) {
+  remove(@Param('id') id: string) {
```

Comment:
@BULMA-OO You have missed adding return types for all the function in this service.